pkg/cmd/cli/describe/describer.go

```diff
@@ -377,7 +377,11 @@ func (d *BuildConfigDescriber) Describe(namespace, name string) (string, error)
 	if err != nil {
 		return "", err
 	}
-	buildList, err := d.Builds(namespace).List(labels.SelectorFromSet(labels.Set{buildapi.DeprecatedBuildConfigLabel: name}), fields.Everything())
+	buildConfigLables := labels.Set{
+		buildapi.BuildConfigLabel: name,
```

Comment:
why not?  the whole point here is that our code should be able to handle:
1) old build that only have the old label
2) newer builds that have both labels (current state)
3) future builds that only have the new label (future state once we remove the deprecated label entirely)